src/frontend/screens/ObservationsList/index.tsx

```diff
@@ -91,6 +89,22 @@ export const ObservationsList: NativeNavigationComponent<'ObservationList'> = ({
   );
 };
 
+export function CreateNavigationOptions(): BottomTabNavigationOptions {
```

Comment:
nit: to be consistent (also this is a normal function, not a component to being uppercase isn't strictly necessary)

```suggestion
export function createNavigationOptions(): BottomTabNavigationOptions {
```

Reply:
kind of surprised doing `useIntl()` here works. i would suggest following the approach that existing implementations use though

Reply:
this is actually making me realize that i am using a hook in it...

Reply:
https://github.com/digidem/comapeo-mobile/pull/309/commits/56f1954b6b4c573fbd9b1d31554ea06a2969ec46